cdap-security/src/main/java/co/cask/cdap/security/server/ExternalAuthenticationServer.java

```diff
@@ -237,11 +238,9 @@ protected void initHandlers() throws Exception {
   protected Executor executor(State state) {
     final AtomicInteger id = new AtomicInteger();
     //noinspection NullableProblems
-    final Thread.UncaughtExceptionHandler h = new Thread.UncaughtExceptionHandler() {
-      @Override
-      public void uncaughtException(Thread t, Throwable e) {
-      }
-    };
+
+    final Thread.UncaughtExceptionHandler h = new UncaughtExceptionHandler();
```

Comment:
Same comment as above. Should set the default in `AuthenticateServerMain` instead and get rid of this.